themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,34 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group']): ?>
+                    <?php if (isset($this->config->AuthorResultListInfo->ResultListPage) && ($this->config->AuthorResultListInfo->ResultListPage == 'Short')): ?>
+                        <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+			<?php $authorCount = count($authors) ?>
+		        <?php $authorNameLimit = $this->config->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                        <?php if (isset($authors) && !empty($authors)): ?>
```

Comment:
`empty` includes a built-in `isset` check, so you don't need to separately call `isset` here.

Reply:
ahh good to know, thanks Demian. That's included in my latest follow-up